test/files/neg/wildcards-future.check

```diff
@@ -1,10 +1,10 @@
 wildcards-future.scala:7: error: type mismatch;
- found   : Map[_$1,_$2] where type _$2 >: Null, type _$1 <: AnyRef
+ found   : scala.collection.immutable.Map[_$1,Any] where type _$1 <: AnyRef
  required: Map[String,String]
   underscores : Map[String, String] // error wildcard variables starting with `_`
   ^
 wildcards-future.scala:9: error: type mismatch;
- found   : Map[?$1,?$2] where type ?$2 >: Null, type ?$1 <: AnyRef
+ found   : scala.collection.immutable.Map[?$1,Any] where type ?$1 <: AnyRef
  required: Map[String,String]
```

Comment:
The before looks better, no?  We're dealiasing more now?

Reply:
Same reason as with `Either` - `Map` is covariant in the value parameter so it's widened to `Any`

Reply:
scala.Map is being dealiased to scala.collection.immutable.Map only for the "found" - that's the part I'm saying looked better before.

Reply:
Agree this particular example is a small regression, but probably just due to existing behavior in type / message printing?

Reply:
I think it's because `existentialAbstraction` calls `normalizeAliases`